src/main/java/core/basesyntax/service/impl/FruitTransactionParserServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.OperationService;
+import core.basesyntax.service.TransactionParserService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitTransactionParserServiceImpl implements TransactionParserService {
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static final String COMA_SPLITTER = ",";
+    private static final int INDEX_OF_OPERATIONS_TYPE = 0;
+    private static final int INDEX_OF_FRUIT_TYPE = 1;
+    private static final int INDEX_OF_QUANTITY = 2;
+    private static final int INDEX_OF_BEGINNING_OF_TRANSANTIONS = 1;
+    private final OperationService operationService;
+    private final List<FruitTransaction> transactions;
```

Comment:
Why do you need to store transactions list as a field?

Reply:
> Why do you need to store transactions list as a field?

I created it as a field because creating a list is the main purpose of this class and maybe we need to do something else with this list. Maybe it's redundant